variables.tf

```diff
@@ -75,9 +75,11 @@ variable "enable_private_default_route" {
 variable "flow_logs" {
   type = object({
     iam_role_name     = string
+    iam_role_boundary_policy = string
```

Comment:
It seems you didn't format your code using `terraform fmt`

Reply:
Commited changes as requested.